src/store/personViews.js

```diff
@@ -414,6 +414,23 @@ export default function personViews(state = null, action) {
             dirtyPersonIds.push(action.meta.rowId);
             affectedViewIds.push(action.meta.viewId);
         }
+        else if (action.type == types.UPDATE_PERSON + '_FULFILLED') {
+            const pId = action.payload.data.data.id
+            dirtyPersonIds.push(pId);
+
+            Object.keys(state.rowsByView)
+                .filter(viewId => {
+                    // Filter views that contain the dirty person object
+                    const rowList = state.rowsByView[viewId];
+                    if (rowList.items) {
+                        return rowList.items.find(row => row.data.id === pId)
+                    }
+                })
```

Comment:
The corresponding logic for other actions don't just check for views containing a person, but views that may be affected by the update. In the case of a taging for example, on lines 355-377, it looks for views that contain a tag column.

Because what we're working with here is a person object changing, any view may have updated in response to this. So I think there should be no filtering happening here at all. The next pass starting at line 437 will go through all of the views and flag them if the person exists either in a saved row, or in a row retrieved dynamically using a query.

Reply:
Just to clarify: When I say that "any view" will have been affected, that's not strictly true, because not all views necessarily contain a `person_field` column. But in practice, they always do, because Zetkin Organize adds first name and last name by default, and it's extremely rare that someone would delete them and not add any other `person_field` columns.